drivers/built_for_zero_report/app/models/built_for_zero_report/calculator.rb

```diff
@@ -111,29 +111,29 @@ def average_lot_to_housing
 
     # @return [SourceDataHash] clients in cohort who became inactive in the reporting period
     def inactive
-      source_data.select { |_, v| v[:change] == 'destroy' && v[:reason] == 'Inactive' }
+      source_data.select { |_, v| v[:change] == 'destroy' && v[:reason].downcase == 'Inactive'.downcase }
```

Comment:
Maybe just do the downcase directly?
```suggestion
      source_data.select { |_, v| v[:change] == 'destroy' && v[:reason].downcase == 'inactive' }
```

Reply:
Or, maybe, better yet, extract constants in `GrdaWarehouse::SystemCohorts::Base` and use them instead of the strings?
```suggestion
      source_data.select { |_, v| v[:change] == 'destroy' && v[:reason].downcase == ::GrdaWarehouse::SystemCohorts::Base::INACTIVE_CLIENT.downcase }
```

(Would also require using the constants when cohort changes are recorded.